### DISCUSSION TITLE
Tell me your favorite programming language(好きなプログラミング言語を教えてください)

### DISCUSSION THREAD

**Discussion by uriuriboo** [NAIST-SE/AttractingContributors#420] [General]
あなたの好きなプログラミング言語を教えてください。
リスト以外のでも大丈夫です。

https://github.com/uriuriboo/AttractingContributors

**Comment by uriuriboo** [comment]
リスト以外としていますがリストに追加する形でお願いします

**Comment by aru1702** [comment]
CとかC#とかJava興味があります。
uriuribooさんはどうですか？

See my repo: https://github.com/aru1702/AttractingContributors-TouhouFumo
Hop into my discussion: https://github.com/NAIST-SE/AttractingContributors/discussions/407

**Reply by uriuriboo** [reply]
どの言語もいいですね。
私はRust、Juliaあたりに興味があります。Rustはかなり難しく使うのに苦労していますが。
